### PR TITLE
Improvements: Table refactoring & Fix 'any' type in web

### PR DESCRIPTION
**Pull request by fruneen**

### CODE REVIEW

File: template/apps/web/src/config/index.ts
Comment (fruneen) [inline template/apps/web/src/config/index.ts:1]: fixing of cycle deps

File: template/apps/web/src/services/api.service.ts [hunk 1]
```diff
@@ -1,14 +1,21 @@
-// eslint-disable-next-line max-classes-per-file
 import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
 
 import config from 'config';
 
+type ApiErrorHandler = (error: ApiError) => void;
+
+type EventHandler = ApiErrorHandler;
```
Comment (fruneen) [inline template/apps/web/src/services/api.service.ts:7]: if another handler should be added, developer should update `EventHandler` type adding own handler type

File: template/apps/api/src/resources/token/token.service.ts [hunk 2]
```diff
@@ -34,15 +37,14 @@ const createAuthTokens = async ({ userId, isShadow }: { userId: string; isShadow
...
-  return (
-    tokenEntity && {
-      userId: tokenEntity.userId,
-      isShadow: tokenEntity.isShadow,
-    }
-  );
+  return tokenEntity;
 };
 
 const removeAuthTokens = async (accessToken: string) => service.deleteMany({ value: { $in: [accessToken] } });
```
Comment (ilyevskii) [inline template/apps/api/src/resources/token/token.service.ts]:
```suggestion
const removeAuthTokens = async (accessToken: string) => service.deleteMany({ value: accessToken } });
```

File: template/apps/api/src/resources/token/token.service.ts [hunk 3]
```diff
@@ -6,17 +6,20 @@ import { DATABASE_DOCUMENTS } from 'app-constants';
 import { tokenSchema } from 'schemas';
 import { Token, TokenType } from 'types';
 
+type TokenPayload = Pick<Token, 'userId' | 'isShadow'> & { tokenType: Token['type'] };
```
Comment (ilyevskii) [inline template/apps/api/src/resources/token/token.service.ts]:
```suggestion
type TokenPayload = Pick<Token, 'userId' | 'isShadow'> & { tokenType: TokenType };
```

File: template/apps/web/src/components/Table/LoadingState/index.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+interface LoadingStateProps extends StackProps {
+  skeletonProps?: SkeletonProps;
+  rowsCount?: number;
+}
+
+const LoadingState: FC<LoadingStateProps> = ({ rowsCount = 3, skeletonProps, ...rest }) => (
+  <Stack gap="md" {...rest}>
+    {Array.from(Array(rowsCount).keys()).map((i) => (
+      <Skeleton key={i} h={60} radius="sm" {...skeletonProps} />
+    ))}
```
Comment (ilyevskii) [inline template/apps/web/src/components/Table/LoadingState/index.tsx]:
```suggestion
    {Array.from(Array(rowsCount), (i) => (
      <Skeleton key={i} h={60} radius="sm" {...skeletonProps} />
    ))}
```

**Review by ilyevskii — APPROVED** [review]

**Pull request merged**